docs/design/lexical_conventions/comments.md

```diff
@@ -0,0 +1,156 @@
+# Comments
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Comments](#comments)
```

Comment:
We use pre-commit for toc generation, amongst other things. Note the errors at https://github.com/carbon-language/carbon-lang/actions/runs/4296541596/jobs/7681833240

You can run pre-commit locally following the instructions at https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#running-pre-commit

Reply:
I removed the toc from both files and did the commit, and it showed the below and updated both toc:
![image](https://user-images.githubusercontent.com/72661784/224465908-293211db-64b9-4632-8fe2-099ae5a24af0.png)

Is it fine?

Reply:
There are still pre-commit issues. You can see them in the CI run I just enabled, and as Jon mentioned you can reproduce these (and even fix them!) locally by using the `pre-commit` tool according to the docs Jon linked.

Reply:
Can you pls check [this](https://github.com/carbon-language/carbon-lang/pull/2641#discussion_r1133491316)